Producer/src/JetsFiller.cc

```diff
@@ -325,6 +363,62 @@ JetsFiller::setRefs(ObjectMapStore const& _objectMaps)
     }
   }
 
+  // Set the references to the secondary vertices
+  if (!csvTag_.empty()) {
+    VertexDistance3D vdist;
+
+    float maxScore(0);
+    float maxSignificance(0);
+
+    auto& jetMap(objectMap_->get<reco::Jet, panda::Jet>());
+    auto& svMap(_objectMaps.at("secondaryVertices").get<reco::VertexCompositePtrCandidate, panda::SecondaryVertex>());
+    auto& pvMap(_objectMaps.at("vertices").get<reco::Vertex, panda::RecoVertex>());
+
+    reco::Vertex pv;
```

Comment:
Do we need to copy the vertex by value here, or can this be `reco::Vertex const* pv(0);` ?

Reply:
I don't know much about edm::Ptr, but let me know if this is the wrong way to handle it: https://github.com/PandaPhysics/PandaProd/pull/49/commits/6d45403b200d9a16f63e25c4a73dc19d98cafe7f